src/Utils.php

```diff
@@ -300,13 +315,13 @@ public static function jsonEncode($value, int $options = 0, int $depth = 512): s
      * Wrapper for the hrtime() or microtime() functions
      * (depending on the PHP version, one of the two is used)
      *
-     * @return float|mixed UNIX timestamp
+     * @return float UNIX timestamp
      *
      * @internal
      */
     public static function currentTime()
```

Comment:
if we are sure that this returns a float.. we could remove the docblock and add a return type.. 

or leave the docblock but I don't see much value to `@return float UNIX timestamp` instead I would prefer if it could be moved to description.

Reply:
Adding the return type on a public method would be a BC break

Reply:
yes.. but we are moving from 6 towards 7.. and v7 is still as beta.. so we are allowed to do so.

Reply:
also this is a function marked with `@internal`.. which means that it is not under our BC policy.

Reply:
Great! Fixed in 7f77c9ec0be33951e751f0a3394f908c6aafd880

Planning on adding more typehints? This PR focuses on fixing the PHPStan errors, so I'd be happy to help on adding more typehints / code safety in another PR.